plugin/pkg/scheduler/algorithm/priorities/priorityutil/non_zero.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
BTW - would it be possible to move this change to a separate PR? (there seems to be an independent commit)